socorro/unittest/external/es/base.py

```diff
@@ -774,6 +794,15 @@ def test_with_version(self):
     return decorated
 
 
+class SuperSearchWithFields(SuperSearch):
```

Comment:
Like I alluded to, it's a personal choice. Like PHP vs Go.
Also, you have to excuse me because I was on a high of reducing
CurrentProducts, CurrentVersions and ProductsVersions down to just one
thing (...but used differently). :)

On Monday, March 21, 2016, Adrian Gaudebert <notifications@github.com>
wrote:

> In socorro/unittest/external/es/base.py
> <https://github.com/mozilla/socorro/pull/3242#discussion_r56846125>:
>
> > @@ -774,6 +794,15 @@ def test_with_version(self):
> >      return decorated
> >
> >
> > +class SuperSearchWithFields(SuperSearch):
>
> I actually think it's simpler to have this class in the base.py file. To
> me it's the same it terms of instant readability (I'm not used to the
> functools module, so I would have to go read the doc of partial anyway).
> And with my class, each file using SuperSearch only has one import to do
> instead of 3 (functools, SuperSearch and SuperSearchFields).
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/socorro/pull/3242/files/7d6d6a3594d03be0b32aaa2eaf0b750211389f0e#r56846125>
>


-- 
Peter Bengtsson
Mozilla Tools & Services